packages/web-components/src/components.d.ts

```diff
@@ -271,7 +271,7 @@ export namespace Components {
          */
         "icon": string;
         /**
-          * The icon SVG's title attribute. Used for accessibility. If none is provided, the icon name will be used.
+           * The icon SVG's title attribute. Used for accessibility. If none is provided, the icon name will be used.
```

Comment:
Are you running a `npm run build.stencil` at all? That command will generate this file. I can't remember if I've asked you that already, so sorry in advance.

Reply:
Not that I know of! When I clone the repo and branch it the file is already there. :) https://github.com/RocketCommunicationsInc/astro/tree/next/packages/web-components/src
It seems that running start.stencil modifies it.